src/Interactive/EditorFeatures/Core/Implementation/Interactive/InertClassifierProvider.cs

```diff
@@ -16,8 +17,7 @@ namespace Microsoft.CodeAnalysis.Editor.Implementation.Interactive
     /// buffers that have been reset but which we still want to look good.
     /// </summary>
     [Export(typeof(IClassifierProvider))]
-    [ContentType(ContentTypeNames.RoslynContentType)]
-    [TextViewRole(PredefinedTextViewRoles.Document)]
+    [TextViewRole(PredefinedInteractiveTextViewRoles.InteractiveTextViewRole)]
```

Comment:
If you `#reset`, what happens to previous submissions with this change?  Do they stay classified?